Jellyfin.Server.Implementations/Users/DefaultPasswordResetProvider.cs

```diff
@@ -119,7 +113,7 @@ public async Task<ForgotPasswordResult> StartForgotPasswordProcess(User user, bo
 
             await using (FileStream fileStream = File.OpenWrite(filePath))
             {
-                _jsonSerializer.SerializeToStream(spr, fileStream);
+                fileStream.Write(JsonSerializer.SerializeToUtf8Bytes(spr));
```

Comment:
`Serialize` to the stream directly instead of -> bytes -> stream

Reply:
🤦 don't know why I didn't use the async methods in the first place.